tic1.js

```diff
@@ -0,0 +1,127 @@
+// this is a test
+const readline = require('readline');
+
+const readlineInterface = readline.createInterface({
+  input: process.stdin,
+  output: process.stdout
+});
+
+function ask(questionText) {
+    return new Promise((resolve, reject) => {
+      readlineInterface.question(questionText, resolve);
+    });
+  }
+
+let cell1 = 1
+let cell2 = 2
+let cell3 = 3
+let cell4 = 4
+let cell5 = 5
+let cell6 = 6
+let cell7 = 7
+let cell8 = 8
+let cell9 = 9
+let cells = [cell1, cell2, cell3, cell4, cell5, cell6, cell7, cell8, cell9]
+let divider = '--- --- ---'
+
+let printGrid = function() {
+    console.log(('\n '+cells[0]+' | '+cells[1]+' | '+cells[2]+'\n'+divider+
+    '\n '+cells[3]+' | '+cells[4]+' | '+cells[5]+'\n'+divider+
+    '\n '+cells[6]+' | '+cells[7]+' | '+cells[8]))
+    }
+
+let askForInputX = async function(){

```

Comment:
`askForInputX` is very similar to `askForInputO`. Can you extract the common elements into a separate function?

Reply:
cf. https://en.wikipedia.org/wiki/Don%27t_repeat_yourself